fmt/src/visit.rs

```diff
@@ -334,6 +314,11 @@ impl Visitable for SourceUnitPart {
             SourceUnitPart::VariableDefinition(variable) => v.visit_var_definition(variable),
             SourceUnitPart::TypeDefinition(def) => v.visit_type_definition(def),
             SourceUnitPart::StraySemicolon(_) => v.visit_stray_semicolon(),
+            SourceUnitPart::DocComment(doc) => v.visit_doc_comment(doc),
+            SourceUnitPart::Using(_using) => {
+                // TODO implement me
+                Ok(())
```

Comment:
should this log `trace::warn!("Formatting not yet implemented for Using")` or something? Same for all our other unimplemented stuff?